nativeVsLodash.js

```diff
@@ -109,9 +109,6 @@ const a0 = [1];
 concatSuite.add('lodash', () => _.concat(a0, 2, [3], [
     [4]
   ]))
-  .add('underscore', () => __.concat(a0, 2, [3], [
```

Comment:
It's true that there weren't any definition of `__` which should have been the case, so this doesn't need to be removed.